musicbot/bot.py

```diff
@@ -1423,7 +1423,7 @@ def check(m):
 
             player.skip()  # check autopause stuff here
             await self._manual_delete_check(message)
-            return
+            return Response("Your request to skip " + str(player.current_entry.title) + " has passed.",reply=True,deleteafter=20)
```

Comment:
This would be better done using a `.format()`. Spaces after the commas are good practice too, and `deleteafter` should actually be `delete_after`